Makefile

```diff
@@ -12,14 +12,14 @@ extra_quality_checks:
 
 # this target runs checks on all files
 quality:
-	black --required-version 23 --check $(check_dirs)
 	ruff $(check_dirs)
+	ruff format --check $(check_dirs)
```

Comment:
If it's a truly 1:1 then this should come first, what's the reason it's after? :)

Reply:
For instance the Ruff `COM` rules (which are not enabled at present, but could be) may add trailing commas, and since Ruff format supports the "magic trailing comma" convention Black uses (i.e. if e.g. an arglist has a trailing comma, the whole shebang should be split into line-per-argument), that formatting needs to happen after a trailing comma is added.